t/test_bhutan.rb

```diff
@@ -0,0 +1,43 @@
+
+require 'csv_to_popolo'
+require 'minitest/autorun'
+require 'json'
+
+describe 'Bhutan' do
+  subject { Popolo::CSV.new('t/data/bhutan.csv') }
+
+  let(:ppl)  { subject.data[:persons] }
+  let(:orgs) { subject.data[:organizations] }
+  let(:mems) { subject.data[:memberships] }
+  let(:legm) { mems.select { |m| m[:role] == 'member' } }
+
+  let(:pm) { ppl.find { |i| i[:id] == 'person/1'  } }
```

Comment:
Prefer `detect` over `find`.<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.